src/main/java/com/ahoo/issuetrackerserver/member/MemberController.java

```diff
@@ -17,8 +21,22 @@ public class MemberController {
     private final MemberService memberService;
 
     @PostMapping("/members")
-    @ResponseStatus(HttpStatus.OK)
     public MemberResponse signUpByGeneral(@Valid @RequestBody GeneralMemberCreateRequest memberCreateRequest) {
         return memberService.signUpByGeneral(memberCreateRequest);
     }
+
+    @GetMapping("/members/loginid")
+    public Boolean isDuplicateLoginId(@RequestParam("loginId") String loginId) {
+        return memberService.validateLoginId(loginId);
+    }
```

Comment:
현재 메소드 네이밍은 `이거 중복된 아이디야?`라고 묻고있는데, 중복검사를 통과하여 유효한 아이디인 경우 `true`를 리턴하고 있습니다.
1. 메서드 네이밍을 수정하거나
2. 응답형식을 수정하거나

하면 좋을 것 같아요. 개인적으로 저는 2번을 채택해서 이런 응답값을 내려주는 것이 더 좋을 것 같습니다.
```suggestion
    public ResponseEntity<Void> checkDuplicatedLoginID(@RequestParam("loginId") String loginId) {
        return ResponseEntity.ok().build();
```

Reply:
고민해보니까, 중복 검사에 대한 응답 DTO를 따로 생성하여 에러가 아닌 중복검사 결과를 응답해주는 것이 더 좋을 것 같습니다.

Reply:
> 메서드 네이밍을 수정하거나, 응답형식을 수정하거나

FE, iOS분들과 협의한 결과 Boolean 타입의 응답만 내려주면 되는 것으로 확정되어 중복검사 결과 메시지는 회원가입 API에서만 활용하도록 수정했습니다.
따라서 메서드 네이밍을 수정하는 방식을 채택했습니다.